engine/src/cmd/unit_factory.cpp

```diff
@@ -1,238 +0,0 @@
-                                             int faction,
-                                             const string &modifications,
-                                             Flightgroup *flightgrp,
-                                             int fg_subnumber )
-{
-    return new GameEnhancement( filename,
-                                faction,
-                                modifications,
-                                flightgrp,
-                                fg_subnumber );
-}
-
-Building* UnitFactory::createBuilding( ContinuousTerrain *parent,
-                                       bool vehicle,
-                                       const char *filename,
-                                       bool SubUnit,
-                                       int faction,
-                                       const std::string &unitModifications,
-                                       Flightgroup *fg )
-{
-    return new GameBuilding( parent,
-                             vehicle,
-                             filename,
-                             SubUnit,
-                             faction,
-                             unitModifications,
-                             fg );
-}
-
-Building* UnitFactory::createBuilding( Terrain *parent,
-                                       bool vehicle,
-                                       const char *filename,
-                                       bool SubUnit,
-                                       int faction,
-                                       const std::string &unitModifications,
-                                       Flightgroup *fg )
-{
-    return new GameBuilding( parent,
-                             vehicle,
-                             filename,
-                             SubUnit,
-                             faction,
-                             unitModifications,
-                             fg );
-}
-
-Asteroid* UnitFactory::createAsteroid( const char *filename,
-                                       int faction,
-                                       Flightgroup *fg,
-                                       int fg_snumber,
-                                       float difficulty )
-{
-    Asteroid *ast = new GameAsteroid( filename,
-                                      faction,
-                                      fg,
-                                      fg_snumber,
-                                      difficulty );
-
-    return ast;
-}
-
-
-Terrain* UnitFactory::createTerrain( const char *file, Vector scale, float position, float radius, Matrix &t )
-{
-    Terrain *tt;
-    tt = new Terrain( file, scale, position, radius );
-    tt->SetTransformation( t );
-    return tt;
-}
-
-ContinuousTerrain* UnitFactory::createContinuousTerrain( const char *file, Vector scale, float position, Matrix &t )
-{
-    ContinuousTerrain *ct;
-    ct = new ContinuousTerrain( file, scale, position );
-    ct->SetTransformation( t );
-    return ct;
-}
```

Comment:
I guess these weren't being used anywhere, were they?

Reply:
Sure looks that way. The current state of the code is such that a lot of stuff is either obsolete or in preparation for something the original dev didn't get to do. It's very frustrating and hard to understand, as there's no spec of issues documenting it anymore.